dappnode_package.json

```diff
@@ -23,7 +23,7 @@
   "dependencies": {
     "bind.dnp.dappnode.eth": "0.2.11",
     "ipfs.dnp.dappnode.eth": "0.2.23",
-    "dappmanager.dnp.dappnode.eth": "0.2.93",
+    "dappmanager.dnp.dappnode.eth": "0.2.95",
     "wifi.dnp.dappnode.eth": "0.2.9"
   },
   "optionalDependencies": {
```

Comment:
Im confused, in no situation is a VPN client NOT installed.  Wireguard is a mandatory package, VPN is optional and any install in the last 2 years or so, its not included.  WiFi is more optional than a VPN client. 

 
But the biggest thing is that HTTPS is not ANYWHERE in core deps or in core optional deps.  Its causing problems and im getting it uninstalled or not updated randomly.

Reply:
> Im confused, in no situation is a VPN client NOT installed. Wireguard is a mandatory package, VPN is optional and any install in the last 2 years or so, its not included
The core packges to be installed are determined here https://github.com/dappnode/DAppNode/blob/d01ecfc2c51d4bbb7337097f8bd622b7d615ff60/scripts/dappnode_install.sh#L65
Installing dappnode should always come with a pre-installed vpn package (either openvpn or wireguard). 

> WiFi is more optional than a VPN client.
Wifi is a must in physical dappnodes. Generally, Its a better plugNplay solution

> But the biggest thing is that HTTPS is not ANYWHERE in core deps or in core optional deps. Its causing problems and im getting it uninstalled or not updated randomly.

HTTPS is not a mandatory core package but optional. Also, optional deps means "if installed in dappnode, they will be updated to defined version". Check out https://github.com/dappnode/DNP_DAPPMANAGER/blob/904843640972cf669b4b4e5e22829d0d81799a19/packages/schemas/src/schemas/manifest.schema.json#L201